skyportal/tests/frontend/test_groups.py

```diff
@@ -67,9 +67,11 @@ def test_add_new_group_user_admin(
     el_input.clear()
     ActionChains(driver).move_to_element(el_input).click().send_keys(
         user_no_groups.username
-    ).pause(2).send_keys(Keys.ENTER).perform()
+    ).pause(5).send_keys(Keys.ENTER).perform()
     driver.click_xpath('//input[@type="checkbox"]')
     driver.click_xpath('//input[@value="Add user"]')
+
+    driver.save_screenshot('test_add_new_group_user_admin.png')
```

Comment:
This was probably for debugging. Same above.

Reply:
fixed